_security/configuration/best-practices.md

```diff
@@ -0,0 +1,122 @@
+
+Whenever possible adhere to these recommendations:
+- `audit.log_request_body: false`
+- `audit.resolve_bulk_requests: false`
+- `compliance.write_log_diffs`
+- As few as possible entries for `compliance.read_watched_fields`
+- As few as possible entries for `compliance.write_watched_indices`
+
+## 7. Explore disabling private tenant
+
+In many cases the use of private tenants is not needed, however the feature is switched on by default. As a result every OpenSearch Dashboards user is provided with their own private tenant and therefore new index to save objects. This can add up to large number of unnecessary indexes. Determine if this feature is indeed needed and it isn't, disable it with the following configuration in `config.yml` file:
+```
+config:
+  dynamic:
+    kibana:
+      multitenancy_enabled: true
+      private_tenant_enabled: false
+```
+
+## 8. Manage the configuration via securityadmin.sh
+
+We recommend to use securityadmin.sh to manage configuration of your clusters, `securityadmin.sh` is a command-line tool provided by OpenSearch for managing security configurations. It allows administrators to efficiently manage security settings, including roles, role mappings, and other security-related configurations, within an OpenSearch cluster.
+
+Changes with made with `securityadmin.sh` should be preceded by a backup of the current configuration, otherwise configuration can be lost! For more detailed information visit the [security admin configuration](https://opensearch.org/docs/latest/security/configuration/security-admin/) section in the documentation. 
+
+
+### Benefits of using securityadmin.sh:
+1. Consistency: By using securityadmin.sh, administrators can ensure consistency across security configurations within the cluster. This helps maintain a standardized and secure environment.
+2. Automation: securityadmin.sh enables automation of security configuration tasks, making it easier to deploy and manage security settings across multiple nodes or clusters.
+3. Version Control: Security configurations managed through securityadmin.sh can be version-controlled using standard version control systems like Git. This facilitates tracking changes, auditing, and reverting to previous configurations if necessary.
+
+To prevent any configuration overrides, it is important to note that configuration changes made through the OpenSearch Dashboards UI or using the OpenSearch API provide a convenient way to modify security configurations. To ensure updates made in this way are maintained through the securityadmin.sh tool, a backup of configuration should first be made to ensure all configuration are captured prior to uploading the modified configuration using securityadmin.sh
```

Comment:
suggestion: 
```
To prevent configuration overrides, it's important to note that changes made through the OpenSearch Dashboards UI or using the OpenSearch API provide a convenient way to modify security settings. To ensure these updates are preserved when using the `securityadmin.sh` tool, first create a backup of the current configuration. This ensures all configurations are captured before uploading the modified configuration with `securityadmin.sh`.
```